rfcs/20230210-upper-snake-case-enum-cases.md

```diff
@@ -0,0 +1,62 @@
+# Upper-snake case enum cases
```

Comment:
Isn't the casing normally just called `UPPER_CASE`? 🤔

Reply:
I couldn't find a good reference to this. 🤷🏻 I assumed upper-case would mean `UPPER CASE` 🤔

Reply:
Ah right yeah, let's keep the wording `UPPER_SNAKE`, but write it like that :) 

I often prefer writing the casing names in the casing they are, to make it very clear 👍